pkg/subctl/operator/brokercr/ensure.go

```diff
@@ -38,15 +39,20 @@ func Ensure(config *rest.Config, namespace string, brokerSpec submariner.BrokerS
 		Spec: brokerSpec,
 	}
 
-	dynClient, err := dynamic.NewForConfig(config)
+	client, err := submarinerClientset.NewForConfig(config)
 	if err != nil {
 		return err
 	}
 
-	client := dynClient.Resource(schema.GroupVersionResource{
-		Group:    submariner.SchemeGroupVersion.Group,
-		Version:  submariner.SchemeGroupVersion.Version,
-		Resource: "brokers"}).Namespace(namespace)
-
-	return util.CreateAnew(client, brokerCR, nil)
+	return util.CreateAnew(&resource.InterfaceFuncs{
```

Comment:
CreateAnew? :) it isn't very descriptive, a client? :)

Reply:
just opinion, but wouldn't it be more "humanized" to say:

util.CreateANew(&broker, .....) ?

Reply:
Not sure what you mean by "humanized"...  so you propose to capitalize the N? `Anew` is one word. Also this function is in admiral so renaming it would be outside the scope of this PR.

Reply:
@mangelajo :point_up: